### PR TITLE
Support trace directory to execute test in batch 

### LINKED ISSUE #138: Support directories of traces in `test trace`

### PR DESCRIPTION
**Pull request by rnbguy**
Closes #138

### CODE REVIEW

File: atomkraft/test/__init__.py [hunk 1]
```diff
@@ -37,15 +48,18 @@ def RequiredFileOption(help, default):
 def trace(
     # currently, require the trace to be present.
     # later, there will be an option to pick up the last one from the model
-    trace: Optional[Path] = FileOption("trace to execute", "model"),
+    trace: Optional[Path] = PathOption(
```
Comment (andrey-kuprianov) [inline atomkraft/test/__init__.py]: I believe we agreed to name this option `--path`... `--trace` was fine for a single file; for a directory this name is confusing

**Review by andrey-kuprianov — CHANGES_REQUESTED** [review]
Looks good! only a small renaming is needed

File: atomkraft/test/__init__.py (hunk 1 above)
Comment (rnbguy) [reply atomkraft/test/__init__.py]: Renamed `--trace` to `--path` :slightly_smiling_face:

**Review by andrey-kuprianov — APPROVED** [review]
Thanks for the updates, Rano!

**Pull request merged**